db/post_migrate/20200602034543_make_invite_user_id_redeemed_at_read_only.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class MakeInviteUserIdRedeemedAtReadOnly < ActiveRecord::Migration[6.0]
```

Comment:
Sorry one more comment here. The migration name should reflect the migration of data here instead of marking the columns as readonly. Marking the columns as readonly is the side effect of having to migrate the data.